app/lib/proxy_config_affecting_changes.rb

```diff
@@ -71,7 +71,7 @@ module BackendApiConfigExtension
       after_commit :issue_proxy_affecting_change_events
 
       def issue_proxy_affecting_change_events
-        return unless service
+        return unless service && previous_changes.present?
```

Comment:
I think it is [easier to read](http://www.railstips.org/blog/archives/2008/12/01/unless-the-abused-ruby-conditional/) the following way:
```suggestion
        return if !service || previous_changes.blank?
```

Reply:
Interesting. I find it easier as is now.

Reply:
Oki :+1: Either way for sure not a big deal :+1: approved :green_heart: